mmcv/ops/csrc/pytorch/cpu/pixel_group.cpp

```diff
@@ -3,6 +3,7 @@
 
 #include "pytorch_cpp_helper.hpp"
 #include "pytorch_device_registry.hpp"
+#include <queue>
```

Comment:
why do we need to include `queue` here?

Reply:
Because the implementation need `queue` [here](https://github.com/open-mmlab/mmcv/blob/5efa302db6bc7c1f70d73d8b75d4866123618e9f/mmcv/ops/csrc/pytorch/cpu/pixel_group.cpp#L55).